src/sentry/runner/commands/configoptions.py

```diff
@@ -273,11 +273,16 @@ def sync(ctx: click.Context) -> None:
                                 )
                                 presenter_delegator.flush()
                                 raise
-
                         presenter_delegator.unset(opt.name)
                     else:
-                        presenter_delegator.drift(opt.name, "")
-                        drift_found = True
+                        if (
```

Comment:
Please add tests if you change the logic 
https://github.com/getsentry/sentry/blob/master/tests/sentry/runner/commands/test_configoptions.py